### PR TITLE
Clarify instructions for creation of secrets.

### PR DESCRIPTION
**Pull request by ambercs**
We make sure that files get renamed correctly and better clarify which
values belong in which secret.
Co-authored-by: Zac Bergquist <zbergquist@pivotal.io>

# Why do we need this PR?
We had a hard time figuring out which secrets needed to be created prior to our first deployment. This PR aims to make it a little easier to follow along.

# Changes proposed in this pull request
* ` README.md` updates only

# Reviewer Checklist
- [ ] Code reviewed
- [ ] Topgun tests run
- [ ] Back-port if needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -335,12 +331,42 @@ printf "%s" "$(pbpaste)" > github-client-secret
...
+mv host-key.pub worker/host-key-pub
+mv worker-key.pub worker/worker-key-pub
+mv worker-key worker/worker-key
+
+# web secrets
+mv session-signing-key web/session-signing-key
+mv host-key web/host-key
+cp worker/worker-key-pub web/worker-key-pub
+
+# other concourse secrets (there may be more than the 3 listed below)
```
Comment (taylorsilva) [inline README.md:356]:
```suggestion
# other concourse secrets from the Optional Features section (there may be more than the 3 listed below)
```

**Comment by taylorsilva** [comment]
@ambercs Can you help get the DCO passing by adding the sign-off line to each commit. From the `Details` link on the DCO line: https://github.com/concourse/concourse-chart/pull/98/checks?check_run_id=627693472

**Comment by ambercs** [comment]
Hi @taylorsilva, my apologies, I didn't see your note until last night. Should be all set, please let me know if there is anything else!

**Comment by ambercs** [comment]
@taylorsilva just saw the version bump failure in concourse, is this something that I will need to fix as part of the PR?

**Comment by taylorsilva** [comment]
Nah I'll fix that. Probably gonna remove that check as well 😅

**Pull request merged**